### ISSUE TITLE
Unable to install databricks provider in version 2.4.3 

### ISSUE THREAD

**Issue by srikanthgr** [aws/aws-mwaa-local-runner#213]
I'm trying to install a data bricks provider on a newly created 2.4.3 environment but it fails to install the provider.

Below are the scheduler logs 

`ERROR: Could not find a version that satisfies the requirement aiohttp<4,>=3.6.3 (from apache-airflow-providers-databricks) (from versions: none)`

`ERROR: No matching distribution found for aiohttp<4,>=3.6.3`

I'm using the ./mwaa-local-env package-requirements command to build the whl packages and syncing the same plugins.zip regenerated in the requirements folder. 

requirements.txt which is synced to S3 bucket 

```
--find-links /usr/local/airflow/plugins
--no-index
apache-airflow-providers-databricks==3.3.0
```

requirements.txt which is used to generated the whl file 

```
--constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.4.3/constraints-3.10.txt"

apache-airflow-providers-databricks==3.3.0
```

are there any other changes that need to be done? Any help is much appreciated.

**Comment by lilit-martirosyan-zocdoc** [comment]
@srikanthgr did you fix it?
if yes then how?

**Comment by srikanthgr** [comment]
@lilit-martirosyan-zocdoc yes, I was able to resolve this. So Instead of building the whl files using I used a Linux instance.

**Comment by jhasbani-flipp** [comment]
@srikanthgr can you elaborate a bit more on how you were able to fix this please?

**Comment by srikanthgr** [comment]
@jhasbani-flipp 

Below are the steps I followed to download the whl files:
1. I spinned up a linux EC2 machine and connected it with my local terminal using ssh. 
2. I have followed this documentation to install python 3.10 in EC2 instance [1]
3. Then upgraded the pip using command: "python3.10 -m pip install --upgrade pip" [2]. Then created my requirements file like below using command "vim requirements.txt"
--constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.4.3/constraints-3.10.txt "
apache-airflow-providers-databricks

4. Downloaded the whl files using the command: python3.10 -m pip download -r <PATH TO requirements.txt file>
5. Configured my AWS cli to able to copy my plugins file to s3 using command "aws configure"
6. Copied my plugins file to s3 using command "aws s3 cp plugins.zip s3://path"
7.Then I uploaded my requirements.txt file like below:
--find-links /usr/local/airflow/plugins
--no-index
apache-airflow-providers-databricks

8.And post this I updated my environment and resolved the error.


-=References=-
[1] https://www.gcptutorials.com/post/python-3.10-installation-on-amazon-linux-2  
[2] https://stackoverflow.com/questions/69503329/pip-is-not-working-for-python-3-10-on-ubuntu   

Note: this worked for me for version 2.4.3
